selfdrive/car/ford/carcontroller.py

```diff
@@ -5,10 +5,10 @@
 from openpilot.selfdrive.car.ford import fordcan
 from openpilot.selfdrive.car.ford.values import CarControllerParams, FordFlags
 from openpilot.selfdrive.car.interfaces import CarControllerBase
-from openpilot.selfdrive.controls.lib.drive_helpers import V_CRUISE_MAX
 
 LongCtrlState = car.CarControl.Actuators.LongControlState
 VisualAlert = car.CarControl.HUDControl.VisualAlert
+V_CRUISE_MAX = 145
```

Comment:
why not put into some helper in `selfdrive/car/`? somehow specific to ford?

Reply:
I planned on getting rid of this one, feels like we can set it based on the actuators packet, waiting on Harald looking into this. Will add a TODO to look into this